packages/js/product-editor/src/style.scss

```diff
@@ -13,8 +13,7 @@
 
 @import 'components/radio-field/style.scss';
 @import 'components/iframe-editor/style.scss';
-@import 'components/details-categories-field/style.scss';
```

Comment:
Did you intend to remove these two files for the categories field?

Reply:
Nope.  Error on my part during wonky rebase.  Added back in.